MantidPlot/make_package.rb.in

```diff
@@ -326,6 +326,10 @@ end
 path = "/Library/Python/2.7/site-packages"
 directories = ["sphinx","sphinx_bootstrap_theme","IPython","zmq","pygments","backports", "qtawesome", "qtpy",
                "certifi","tornado","markupsafe","jinja2","psutil","jsonschema","functools32","ptyprocess","CifFile","yaml"]
+
+system("chmod +w #{path}/psutil/_psutil_osx.so")
```

Comment:
I'm not sure I understand why not having write permissions on the source prevents copying?